### ISSUE TITLE
Progressbar text color not suitable for darkmode (Google Colab)

### ISSUE THREAD

**Issue by wanburana** [PyTorchLightning/pytorch-lightning#8447]
## 🐛 Bug

The text showing loss and other metrics is not readable in Google Colab dark mode
![image](https://user-images.githubusercontent.com/66191669/125994821-386974a3-f57e-49e5-9e71-e1b63b5506b0.png)

### To Reproduce
You can use already [**BoringModel**](https://colab.research.google.com/drive/1HvWVVTK8j2Nj52qU4Q4YCyzOm0_aLQF3?usp=sharing) to reproduce the issue
Step 1: Switch from light mode to dark mode
![image](https://user-images.githubusercontent.com/66191669/125995194-581a3bbf-e5c4-4beb-8346-9f3d6572dbe1.png)

Step 2: Run the code until any 'trainer.fit()'  or anything bar related
![image](https://user-images.githubusercontent.com/66191669/125995501-82f6ed67-ceef-4fb1-a1d7-1ffd6a3dec38.png)

### Expected behavior

The text color switch to white instead of staying black

**Comment by SeanNaren** [comment]
I do wonder if other tools such as `rich` have automatic handling of these? cc @tchaton 

otherwise we'll have to detect and do some modification to the standard ProgressBar

**Comment by awaelchli** [comment]
This was reported on the tqdm side here: https://github.com/tqdm/tqdm/issues/1116
not sure if colab's styling is interfering here

**Comment by wanburana** [comment]
UPDATE: I was be able to find the workaround by explicitly changing the text to white
https://github.com/tqdm/tqdm/issues/842#issuecomment-850700218
I assume the problem is mainly from IPython display
Still, it would be nice if we can choose to modify/change progressbar easily. Another problem that I'm facing is that tqdm's progressbar keep switching back and forth from long to short, and the text (batch in epoch, decimal count in metrics) is not properly padded/truncated making it hard to follow since text is jumping around according to the changing of batch's length, metric's decimal length

**Comment by awaelchli** [comment]
I would recommend to close this issue. We are planning to make rich the default bar soon, right @kaushikb11? 
The rich progress bar adapts better to the dark theme:

![image](https://user-images.githubusercontent.com/5495193/144788467-30788707-f4f6-470e-9b8c-dc24613531e9.png)

And it is not something we can fix with tqdm afaik.

**Comment by kaushikb11** [comment]
Yup @awaelchli That's the goal. Wanted to take a stab with Rich's new fix [release](https://github.com/willmcgugan/rich/releases/tag/v10.15.2). But still there are issues with `ddp_cpu`.

Hey @wanburana, you could use `RichProgressBar`, and the default text description for it is `white`, so it will work well with the dark theme. But you wish to customize the color/styles of the components of the `RichProgressBar`, we do provide the utlitity.

```python
trainer = Trainer(..., callbacks=RichProgressBar(theme=RichProgressBarTheme(description="black")))
```
Do check [RichProgressTheme](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/callbacks/progress/rich_progress.py#L166) out to customize styles for various components.

**Comment by kaushikb11** [comment]
@wanburana Seems like it has been fixed with the recent updates to https://github.com/jupyter-widgets/ipywidgets or tqdm.
<img width="1770" alt="Screen Shot 2021-12-06 at 12 34 36 PM" src="https://user-images.githubusercontent.com/45285388/144802543-466c8f48-1547-4cc1-ad77-c4818a544e72.png">

Feel free to reopen this issue if you are facing it again.

**Comment by wanburana** [comment]
@kaushikb11 thank for both of your suggestion. Rich look amazing and i will explore its capability more in the future
